### PR TITLE
Replace CoC enforcement contact with personal addresses

### PR DESCRIPTION
**Pull request by louwers**
This PR replaces the anonymous e-mail address with personal e-mail addresses.

This makes it safer to send a report.

Thanks goes to Nuxt for the idea.

### CODE REVIEW

**Review by HarelM — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -61,8 +61,12 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-[code-of-conduct@maplibre.org](mailto:code-of-conduct@maplibre.org).
+reported to any of the following people:
+
+* Harel Mazor (harel.mazor AT gmail DOT com)
+* Birk Skyum (birk.skyum AT pm DOT me)
+* Oliver Wipfli (oliver.wipfli.f AT gmail DOT com)
```
Comment (wipfli) [inline CODE_OF_CONDUCT.md:68]:
```suggestion
reported to any Governing Board member. For contact details, see https://maplibre.org/about.
```

**Comment by louwers** [comment]
The https://maplibre.org/about page does not list contact details, only LinkedIn, and it is not possible to send a message to someone you are not connected with.

**Comment by wipfli** [comment]
It has slack though, right?

**Comment by louwers** [comment]
Fair enough. Let's put the @maplibre.org e-mail addresses on the website when we have them though.

**Pull request merged**